### DISCUSSION TITLE
CS 학습 이후의 희망 활동

### DISCUSSION THREAD

**Discussion by CloakingGhost** [SeSAC-Next-Level/Algorithm-CodingTest#33] [Ideas]
지금까지 **활동이 어떠했는지** 앞으로 **모임에 대한 의견**을 염두에 두었는지 알고 싶습니다.
따로 말씀주셔도 됩니다.

### 25.01.07(화) 이후 활동
> [!IMPORTANT]
> 주제는 취업, 면접에 국한합니다

1. 기술 면접(CS 학습)
2. 포트폴리오 문답
3. 필살기 발견
4. 그 외

@SeSAC-Next-Level/algorithm 

[참조](https://github.com/SeSAC-Next-Level/Algorithm-CodingTest/wiki/%ED%99%9C%EB%8F%99#cs-%ED%95%99%EC%8A%B5-%EC%9D%B4%ED%9B%84%EC%9D%98-%ED%9D%AC%EB%A7%9D-%ED%99%9C%EB%8F%99-discussion-%EC%9D%B4%EB%8F%99)

**Comment by KoHo512** [comment]
CS 학습 같이 얘기해보니까 훨씬 정리도 잘 되고 기억에 남아서 좋았어요. 다른 활동 하면서 사이사이에 한번씩 복습 겸 심화 학습하면 좋을 거 같습니다!

**Reply by CloakingGhost** [reply]
어떤 내용을 더 심화 학습하면 좋을까요?

**Reply by KoHo512** [reply]
개념이 잡히면 말을 좀 더 정리해서 하는 연습을 하면 좋을 거 같습니다.

**Comment by apricity2u** [comment]
**1. 활동 리뷰**
CS 학습을 통해 정확하게 알지 못했던 개념을 파악하고 정리할 수 있어 좋았습니다. 

**2. 의견** 
다음 주차 스터디 이후 학습한 CS 개념을 총 정리하는 시간이 있으면 좋을 것 같습니다. 서로 지목하고 답하면서 릴레이로 진행해도 재미있을 것 같습니다.

**Comment by devyejin** [comment]
1. 활동 리뷰
- 필살기 발견 : 생각 못 했던 부분이나 보충할 부분을 이야기해 주고 좋았습니다.
- CS 학습 :  학습 방법이나 답변 방향성을 설정할 수 있고 좋았습니다.

2. 의견
- 그동안 했던 미니프로젝트를 정리해 오고 서로 보충할 부분을 이야기해 주면 좋을 것 같습니다.

**Comment by imdla** [comment]
CS 학습 활동을 통해 어떤 방법으로 공부하면 좋을 지 깨달을 수 있었습니다.

다음주 CS 학습 활동 진행 후
1. 일대다 면접처럼 1인당 10~15분 내외 기술 면접을 진행하면 좋을 것 같습니다. 
-> 일대다 면접 형식을 통해 간접적으로 모의 면접을 체험할 수 있고, 다수의 질문을 통해 미처 생각하지 못한 꼬리 물기 질문(심화 질문) 등을 알 수 있을 것 같습니다. 또한, 상대방의 답변을 통해 자신의 부족한 부분을 채우고, 어떤 방향으로 대답하면 좋을지 생각할 수 있는 기회가 될 것 같습니다.

2. CS 학습에 대한 활동 종료 후 포트폴리오나 자기소개서에 대한 활동 진행
-> CS 학습과 달리 개인별 내용이 천차만별이라 1번처럼 시간을 정해(10~15분) 한 명씩 포트폴리오나 자기소개서에서 부족한 부분을 다수의 인원들이 자유롭게 이야기할 수 있는 시간을 가지면 좋겠습니다.

3. 위의 1번 2번을 토대로 -> 자기소개서와 포트폴리오를 보면서 모의 면접 진행 (일대다 형식)
-> 포트폴리오와 자기소개서의 방향성을 통해 질문하고, 그에 기반한 기술 면접으로 탄탄한 서류 및 면접 준비를 할 수 있을 것 같습니다.
일대다 형식으로 진행해 면접에 대한 부담감에 익숙해지고, 면접관 및 면접자가 되어 어떤 예상 질문을 받을지 그에 따른 답변을 어떻게 하면 좋을지 다수의 인원들과 함께 진행해 다양한 생각을 배울 수 있는 시간이 될 것 같습니다.

**Comment by hisunny1007** [comment]
**1. 활동 리뷰**  
- 방대한 CS 지식 중에서 면접에서 나올 수 있는 필수 항목들을 알 수 있어서 좋았습니다.

**2. 의견**  
- 질문에 대한 답변이 충분히 정립될 때 까지 격주로 한 번씩 일대다 기술면접을 진행하면 좋을 것 같습니다.
- 그 외 격주에는 각자 이력서/자기소개서/포폴 등 작성하는 시간이 있으면 좋을 것 같습니다. (모임으로 강제성을 부여하되, 개인 준비 느낌으로)

개별 진행 중에, 공유가 필요한 부분이나 면접 시뮬레이션을 하고 싶으면 의견을 모아 포폴/인성 관련 면접을 진행해도 좋을 것 같습니다.
